saq/types.py

```diff
@@ -93,6 +102,8 @@ class TimersDict(t.TypedDict):
     "How often to clean up stuck jobs in seconds (default 60)"
     abort: int
     "How often to check if a job is aborted in seconds (default 1)"
+    metadata: int
```

Comment:
Why do we need to store it periodically? Isn't it immutable within the worker's lifetime?